bridge/cypress/support/pageobjects/ServicesPage.ts

```diff
@@ -1,8 +1,122 @@
+      .contains(columnName)
+      .click();
+    return this;
+  }
+
+  verifySliBreakdownSorting(columnIndex: number, direction: string, firstElement: string, secondElement: string): this {
+    cy.byTestId('keptn-sli-breakdown')
+      .find('dt-header-row')
+      .first()
+      .find('dt-header-cell')
+      .eq(columnIndex)
+      .find('.dt-sort-header-container')
+      .first()
+      .should('have.class', 'dt-sort-header-sorted');
+    cy.byTestId('keptn-sli-breakdown')
+      .find('dt-header-row')
+      .first()
+      .find('dt-header-cell')
+      .eq(columnIndex)
+      .find('dt-icon')
+      .first()
+      .invoke('attr', 'ng-reflect-name')
+      .should('equal', `sorter2-${direction}`);
+
+    cy.byTestId('keptn-sli-breakdown')
+      .find('dt-row')
+      .eq(0)
+      .find('dt-cell')
+      .eq(columnIndex)
+      .should('have.text', firstElement);
+    cy.byTestId('keptn-sli-breakdown')
+      .find('dt-row')
+      .eq(1)
+      .find('dt-cell')
+      .eq(columnIndex)
+      .should('have.text', secondElement);
+
+    return this;
+  }
+
+  verifySliBreakdown(result: SliResult, isExpanded: boolean): this {
+    cy.byTestId('keptn-sli-breakdown').should('exist');
+    if (isExpanded) {
+      this.assertSliColumn(result.name, 1, 'contain.text', result.name)
+        .assertSliColumn(result.name, 1, 'contain.text', 'Absolute change:')
+        .assertSliColumn(result.name, 1, 'contain.text', 'Relative change:')
+        .assertSliColumn(result.name, 1, 'contain.text', 'Compared with:')
+        .assertSliColumn(
+          result.name,
+          2,
+          'have.text',
+          `${result.value}${result.calculatedChanges?.absolute > 0 ? '+' : '-'}${result.calculatedChanges?.absolute}${
+            result.calculatedChanges?.relative > 0 ? '+' : '-'
+          }${result.calculatedChanges?.relative}% ${result.comparedValue}`
+        );
+    } else {
+      this.assertSliColumn(result.name, 1, 'have.text', result.name)
+        .assertSliColumn(result.name, 1, 'not.contain.text', 'Absolute change:')
+        .assertSliColumn(result.name, 1, 'not.contain.text', 'Relative change:')
+        .assertSliColumn(result.name, 1, 'not.contain.text', 'Compared with:')
+        .assertSliColumn(
+          result.name,
+          2,
+          'have.text',
+          `${result.value} (${result.calculatedChanges?.relative > 0 ? '+' : '-'}${
+            result.calculatedChanges?.relative
+          }%) `
+        );
+    }
+
+    this.assertSliColumn(result.name, 3, 'have.text', result.weight.toString()).assertSliColumn(
+      result.name,
+      7,
+      'have.text',
+      result.score.toString()
+    );
+
+    cy.byTestId(`keptn-sli-breakdown-row-${result.name}`)
+      .find('dt-cell')
+      .eq(2)
+      .find('.error')
+      .should('have.length', result.result == 'pass' ? 0 : 1);
+
+    return this;
+  }
+
+  assertSliColumn(name: string, index: number, assertion: string, value: string): this {
```

Comment:
I will introduce proper uitestid's on the cells and use the sli name instead of index. Also if we add or change the order of columns, the index might always change, you are right.